libraries/manifest-validation/src/validations.ts

```diff
@@ -671,6 +671,21 @@ export const maniTests: Array<Validation> = [
         test: (value: any) => {
             return value && typeof value === "object";
         }
+    },
+    {
+        member: "edge_side_panel",
+        displayString: "Manifest has edge side panel field",
+        infoString: "The edge_side_panel member specifies if your app supports the side panel in the Edge browser.",
+        category: "optional",
+        defaultValue: "",
+        docsLink: "https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/SidePanel/explainer.md",
+        errorString: "edge_side_panel should be a number",

```

Comment:
`edge_side_panel` is an object whereas `edge_side_panel.preferred_width` is a number.